packages/alfa-css/src/properties/overflow/property.ts

```diff
@@ -0,0 +1,39 @@
+import { parse } from "@siteimprove/alfa-lang";
+import { Shorthand } from "../../properties";
+import * as Longhands from "../../properties/longhands";
+import { OverflowGrammar } from "./grammar";
+import { Overflow } from "./types";
+
+/**
+ * @see https://drafts.csswg.org/css-overflow-3/#propdef-overflow
+ */
+export const overflow: Shorthand<"overflowX" | "overflowY"> = {
+  longhands: ["overflowX", "overflowY"],
+  parse(tokens) {
+    let offset = 0;
+    let overflowX: Overflow = Longhands.overflowX.initial();
+    let overflowY: Overflow = Longhands.overflowX.initial();
+    {
+      const { result, position } = parse(tokens, OverflowGrammar, offset);
+
+      if (result !== null) {
+        overflowX = result;
+        offset = position;
+      }
+    }
+
+    {
+      const { result, position } = parse(tokens, OverflowGrammar, offset);
+
+      if (result !== null) {
+        overflowY = result;
+        offset = position;
+      }
+    }
+
+    return {
+      overflowX,
+      overflowY
+    };
+  }
```

Comment:
Getting there! Take a quick look at this section of the CSS syntax spec though: https://www.w3.org/TR/css-syntax/#error-handling. Specifically:

> When interpreting a list of declarations, unknown syntax at any point causes the parser to throw away whatever declaration it’s currently building, and seek forward until it finds a semicolon (or the end of the block). It then starts fresh, trying to parse a declaration again.